ani-cli

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-version_number="4.3.0"
+version_number="4.3.1"
```

Comment:
Actually, don't change the version number on documentation updates

```suggestion
version_number="4.3.0"
```

Reply:
Alright! Done!